jobserver/models.py

```diff
@@ -487,3 +488,37 @@ def repo_name(self):
         if not f.path:
             raise Exception("Repo URL not in expected format, appears to have no path")
         return f.path.segments[-1]
+
+
+class Release(models.Model):
+    id = models.TextField(primary_key=True)
```

Comment:
Can you pop a comment above this line to explain why it's not using Django's AutoField?  I'm assuming this is because you're using the hash as the PK, but future Us will thank Now You!